system_test/aest_perf_SUITE.erl

```diff
@@ -160,6 +154,16 @@ cluster(Names, Spec) -> [spec(N, Names -- [N], Spec) || N <- Names].
 spec(Name, Peers, Spec) ->
     maps:merge(maps:merge(?BASE, Spec), #{name => Name, peers => Peers}).
 
+sync_node(Node, Height, Cfg) ->
+    start_node(Node, Cfg),
+    Start = erlang:system_time(millisecond),
+    wait_for_startup([Node], 0, Cfg),
+    wait_for_sync([Node], Height, Cfg),
+    Block = get_block(Node, Height, Cfg),
+    End = erlang:system_time(millisecond),
+    ?assert(End - Start =< time_to_ms(?cfg(mine_time))),
```

Comment:
I do not manage to follow how this checks relates mine_time.

Reply:
Yeah, thought I should've added a comment to that. It's basically a sanity check so that sync never takes longer than mining.